src/sdltiles.cpp

```diff
@@ -1488,15 +1531,22 @@ static float second_finger_down_x = -1.0f; // in pixels
 static float second_finger_down_y = -1.0f; // in pixels
 static float second_finger_curr_x = -1.0f; // in pixels
 static float second_finger_curr_y = -1.0f; // in pixels
-static unsigned long finger_down_time = 0; // when did the first finger start touching the screen? 0 if not touching, otherwise the time in milliseconds.
-static unsigned long finger_repeat_time = 0; // the last time we repeated input for a finger hold, 0 if not touching, otherwise the time in milliseconds.
-static unsigned long last_tap_time = 0; // the last time a single tap was detected. used for double-tap detection.
-static unsigned long ac_back_down_time = 0; // when did the hardware back button start being pressed? 0 if not touching, otherwise the time in milliseconds.
-static bool is_two_finger_touch = false; // has a second finger touched the screen while the first was touching?
+static unsigned long finger_down_time =
```

Comment:
Please fix this run of indention "fixes".  Move the comments to their own lines instead of breaking the expression.